galasa-parent/dev.galasa.framework.api.ras/openapi.yaml

```diff
@@ -0,0 +1,44 @@
+openapi: 3.0.0
+info:
+  title: Galasa RAS Requests
+  version: 0.0.1
+
+servers:
+  - url: 'http://{server}:{port}'
+    variables:
+      server:
+        default: localhost
+      port:
+        default: "8080"
+
+paths:
+  /ras/{runName}:
```

Comment:
Will need 
GET /ras/runname/{runName}/    this will return all the run ids of all the runs with this name
GET /ras/run/{id}/    this is the internal ID of the run
GET /ras/run/{id}/artifact/{artifactid}   get a specific artifact
GET /ras/run?requestor=xxxx&from=xxxx&to=xxxxx?testclass=xxxxxx?page=xxxx?size=xxxx

Reply:
GET /ras/run/{id}/runlog